### ISSUE TITLE
Autoselect query block for execution

### ISSUE THREAD

**Issue by beeing** [sosedoff/pgweb#482]
Hi, thanks for creating such helpful tool.

One of the features I'd like to see is the ability to auto select query block to execute.

Example:
```sql
SELECT something FROM table
LEFT JOIN table2 ON ....

UPDATE table SET xx=1 WHERE id=1
```

Noticed above we have 2 queries but there's a few empty lines in between.

When the cursor is on SELECT and press Cmd+Enter, it should run the SELECT query only, not all the queries.

This will improve our productivity.

Thanks in advance.

**Comment by sosedoff** [comment]
Hey, thanks for the feedback! I also found myself wanting something like this so there's a big chance a feature like it is going to make it into the next release.

**Comment by sosedoff** [comment]
Should be available in the PR linked above.

**Comment by sosedoff** [comment]
This was implemented in 0.11.11 https://github.com/sosedoff/pgweb/blob/master/CHANGELOG.md#01111---2022-03-29
